ai-platform-unified/notebooks/official/gapic/custom_job_image_classification_model_for_online_prediction.ipynb

```diff
@@ -107,7 +107,7 @@
       },
```

Comment:
<p>You've included outputs for this and the next code cell (! pip install tensorflow).</p><p> </p><p>I don't recall if we explicitly discussed an official for whether or not to include outputs, but I recommend you clear them because:</p><p> </p><ul><li>They contain some (minor) personal information (i.e. the directory structure of your personal machine).</li><li>We haven't committed outputs for the rest of this notebook.</li></ul><p> </p><p>(More generally, I think it's better not to include outputs because they clutter the notebook and their values can change a lot depending on who was the last contributor to run/commit the notebook, leading to unnecessary diffs.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/268/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Good catch, I thought the format tool we used handled this. Apparently, not so let me see what's going on.

Reply:
Fixed @alecglassford !